src/main/java/com/beerjournal/breweriana/item/ItemService.java

```diff
@@ -34,4 +34,21 @@ ItemDto addItem(String ownerId, ItemDto itemDto) {
         return itemDto;
     }
 
+    ItemDto deleteItem(String ownerId, String itemId) {
```

Comment:
Check added for update and delete. But I suppose that it should also be done for adding ;)